Matrix startup times out

I see a lot of logs like this on the deployed services.

```
ESC[36mpfs-goerli_1                  |ESC[0m Traceback (most recent call last):
ESC[36mpfs-goerli_1                  |ESC[0m   File "/usr/local/lib/python3.7/runpy.py", line 193, in _run_module_as_main
ESC[36mpfs-goerli_1                  |ESC[0m     "__main__", mod_spec)
ESC[36mpfs-goerli_1                  |ESC[0m   File "/usr/local/lib/python3.7/runpy.py", line 85, in _run_code
ESC[36mpfs-goerli_1                  |ESC[0m     exec(code, run_globals)
ESC[36mpfs-goerli_1                  |ESC[0m   File "/services/src/pathfinding_service/cli.py", line 122, in <module>
ESC[36mpfs-goerli_1                  |ESC[0m     main(auto_envvar_prefix="PFS")  # pragma: no cover
ESC[36mpfs-goerli_1                  |ESC[0m   File "/usr/local/lib/python3.7/site-packages/click/core.py", line 764, in __call__
ESC[36mpfs-goerli_1                  |ESC[0m     return self.main(*args, **kwargs)
ESC[36mpfs-goerli_1                  |ESC[0m   File "/usr/local/lib/python3.7/site-packages/click/core.py", line 717, in main
ESC[36mpfs-goerli_1                  |ESC[0m     rv = self.invoke(ctx)
ESC[36mpfs-goerli_1                  |ESC[0m   File "/usr/local/lib/python3.7/site-packages/click/core.py", line 956, in invoke
ESC[36mpfs-goerli_1                  |ESC[0m     return ctx.invoke(self.callback, **ctx.params)
ESC[36mpfs-goerli_1                  |ESC[0m   File "/usr/local/lib/python3.7/site-packages/click/core.py", line 555, in invoke
ESC[36mpfs-goerli_1                  |ESC[0m     return callback(*args, **kwargs)
ESC[36mpfs-goerli_1                  |ESC[0m   File "/services/src/raiden_libs/cli.py", line 183, in call_with_blockchain_info
ESC[36mpfs-goerli_1                  |ESC[0m     return callback(**params)
ESC[36mpfs-goerli_1                  |ESC[0m   File "/services/src/raiden_libs/cli.py", line 128, in call_with_common_options_initialized
ESC[36mpfs-goerli_1                  |ESC[0m     return func(**params)
ESC[36mpfs-goerli_1                  |ESC[0m   File "/services/src/pathfinding_service/cli.py", line 107, in main
ESC[36mpfs-goerli_1                  |ESC[0m     service.run()
ESC[36mpfs-goerli_1                  |ESC[0m   File "src/gevent/greenlet.py", line 718, in gevent._greenlet.Greenlet.run
ESC[36mpfs-goerli_1                  |ESC[0m   File "src/gevent/greenlet.py", line 706, in gevent._greenlet.Greenlet._report_error
ESC[36mpfs-goerli_1                  |ESC[0m   File "/services/src/raiden_libs/gevent_error_handler.py", line 24, in error_handler
ESC[36mpfs-goerli_1                  |ESC[0m     Hub.handle_system_error(self, etype, value)
ESC[36mpfs-goerli_1                  |ESC[0m   File "/usr/local/lib/python3.7/site-packages/gevent/hub.py", line 488, in handle_system_error
ESC[36mpfs-goerli_1                  |ESC[0m     self.parent.throw(type, value)
ESC[36mpfs-goerli_1                  |ESC[0m AssertionError: MatrixListener did not start in time.
```

A quick fix in done in #615. However, ideally we should find out what takes so long. A first guess is the joining of all broadcasting rooms.

--- a/deployment/docker-compose.yml
+++ b/deployment/docker-compose.yml
@@ -50,6 +50,8 @@ services:
     labels:
       - "traefik.enable=true"
       - "traefik.frontend.rule=Host: pfs-ropsten-with-fee.services-test.raiden.network"
+    depends_on:
+      - pfs-ropsten
 
   pfs-rinkeby:
     << : *defaults
@@ -60,6 +62,8 @@ services:
     labels:
       - "traefik.enable=true"
       - "traefik.frontend.rule=Host: pfs-rinkeby.services-test.raiden.network"
+    depends_on:
+      - pfs-ropsten-with-fee
 
   pfs-rinkeby-with-fee:
     << : *defaults
@@ -71,6 +75,8 @@ services:
     labels:
       - "traefik.enable=true"
       - "traefik.frontend.rule=Host: pfs-rinkeby-with-fee.services-test.raiden.network"
+    depends_on:
+      - pfs-rinkeby
 
   pfs-goerli:
     <<: *defaults
@@ -81,6 +87,8 @@ services:
     labels:
       - "traefik.enable=true"
       - "traefik.frontend.rule=Host: pfs-goerli.services-test.raiden.network"
+    depends_on:
+      - pfs-rinkeby-with-fee
 
   pfs-goerli-with-fee:
     <<: *defaults
@@ -92,6 +100,8 @@ services:
     labels:
       - "traefik.enable=true"
       - "traefik.frontend.rule=Host: pfs-goerli-with-fee.services-test.raiden.network"
+    depends_on:
+      - pfs-goerli
 
   ms-ropsten:
     <<: *defaults
@@ -139,7 +149,7 @@ services:
       - MSRC_CHAIN_ID=rinkeby
       - MSRC_STATE_DB=/state/ms-rinkeby.db
     depends_on:
-      - ms-rinkeby
+      - msrc-ropsten
 
   msrc-goerli:
     <<: *defaults
@@ -148,7 +158,7 @@ services:
       - MSRC_CHAIN_ID=goerli
       - MSRC_STATE_DB=/state/ms-goerli.db
     depends_on:
-      - ms-goerli
+      - msrc-rinkeby
 
   msrc-goerli-backup:
     <<: *defaults
@@ -159,7 +169,7 @@ services:
       - MSRC_KEYSTORE_FILE=${MS_BACKUP_KEYSTORE_FILE}
       - MSRC_PASSWORD=${MS_BACKUP_PASSWORD}
     depends_on:
-      - ms-goerli-backup
+      - msrc-goerli
 
   builder:
     restart: always
@@ -226,4 +236,4 @@ services:
       - RDN_REGISTRY_SERVICE_URL=""
       - RDN_REGISTRY_ETH_RPC=http://parity.goerli.ethnodes.brainbot.com:8545
     depends_on:
-      - registration-goerli  # The registrations need to run serially
+      - traefik

--- a/src/raiden_libs/logging.py
+++ b/src/raiden_libs/logging.py
@@ -16,7 +16,7 @@ def setup_logging(log_level: str, log_json: bool) -> None:
     logging.basicConfig(level=log_level, stream=sys.stdout, format="%(message)s")
 
     logging.getLogger("web3").setLevel("INFO")
-    logging.getLogger("urllib3").setLevel("DEBUG")
+    logging.getLogger("urllib3").setLevel("INFO")
 
     shared_processors = [
         format_to_hex,

